pkg/console/subresource/configmap/configmap_test.go

```diff
@@ -105,14 +108,49 @@ func TestDefaultConfigMap(t *testing.T) {
 					Finalizers:      nil,
 					ClusterName:     "",
 				},
-				Data:       map[string]string{"console-config.yaml": exampleYaml},
+				Data:       map[string]string{configKey: exampleYaml},
 				BinaryData: nil,
 			},
 		},
 	}
 	for _, tt := range tests {
 		t.Run(tt.name, func(t *testing.T) {
-			if diff := deep.Equal(DefaultConfigMap(tt.args.operatorConfig, tt.args.consoleConfig, tt.args.infrastructureConfig, tt.args.rt), tt.want); diff != nil {
+			cm, _, _ := DefaultConfigMap(tt.args.operatorConfig, tt.args.consoleConfig, tt.args.infrastructureConfig, tt.args.rt)
+
+			// marshall the exampleYaml to map[string]interface{} so we can use it in diff below
+			var exampleConfig map[string]interface{}
+			exampleBytes := []byte(exampleYaml)
+			err := yaml.Unmarshal(exampleBytes, &exampleConfig)
+			if err != nil {
+				t.Error(err)
+			}
+			fmt.Printf("%v\n", exampleConfig)
+
+			// the reason we have to marshall blindly into map[string]interface{}
+			// is that we don't have the definition for the console config struct.
+			// it exists in the console repo under cmd/bridge/config.go and is not
+			// available as an api object
+			var actualConfig map[string]interface{}
+			// convert the string back into a []byte
+			configBytes := []byte(cm.Data[configKey])
+
+			err = yaml.Unmarshal(configBytes, &actualConfig)
+			if err != nil {
+				t.Error("Problem with consoleConfig.Data[console-config.yaml]", err)
+			}
+
+			// compare the configs
```

Comment:
Fixed the test to marshall the configs into `map[string]interface{}` for comparison.  There is still the `nil` setting later to `deep.Equal()` test the rest of the struct, but this ensures everything is properly tested.